### DISCUSSION TITLE
Committing with co-authors to main repository

### DISCUSSION THREAD

**Discussion by yirideveloper** [community/community#49589] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I've created new pull-request and invite the team member but I can't commit to the main repo with him.
I can I do that?

**Comment by Mannyflow7** [comment]
José Pacheco

**Comment by s-spoon** [comment, marked as answer]
It may help you. Take a look this.
https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors
